scripts/wildcards/adjective.txt

```diff
@@ -0,0 +1,209 @@
+Skip to content
```

Comment:
Is this intended?

Reply:
Oh, no, good catch. Only the adjective.txt file I seem to have messed up copy-pasting. Should begin with "beautiful"